app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,14 @@
 class BookingsController < ApplicationController
+
+  def new
+  end
+
+  def create
+  end
+
+  def edit
+  end
+
+  def update
+  end
```

Comment:
Should we include a destroy method? As discussed, we could make it that the cancellation is only permissible 6 hours before the meetup.